File: csharp/DVSClient/DVSClient/email/validate/VerboseOutput.cs
using DVSClient.Common;

namespace DVSClient.Email.Validate
{
    public enum VerboseOutput
    {
        [EnumStringValue("verified")]
        Verified,
        [EnumStringValue("mailboxDisabled")]
        MailboxDisabled,
        [EnumStringValue("mailboxDoesNotExist")]
        MailboxDoesNotExist,
        [EnumStringValue("mailboxFull")]
        MailboxFull,
        [EnumStringValue("syntaxFailure")]
        SyntaxFailure,
        [EnumStringValue("internationalCharactersUnsupported")]
        InternationalCharactersUnsupported,
        [EnumStringValue("unreachable")]
        Unreachable,
        [EnumStringValue("illegitimate")]
        Illegitimate,
        [EnumStringValue("roleAccount")]
        RoleAccount,
        [EnumStringValue("typoDomain")]
        TypoDomain,
        [EnumStringValue("localPartSpamTrap")]
        LocalPartSpamTrap,
        [EnumStringValue("profanity")]
        Profanity,
        [EnumStringValue("disposable")]
        Disposable,
        [EnumStringValue("unknown")]
        Unknown,
        [EnumStringValue("timeout")]
        Timeout,
        [EnumStringValue("acceptAll")]
        AcceptAll,
        [EnumStringValue("relayDenied")]
        RelayDenied,
        [EnumStringValue("BLANK")]
        BLANK
    }
}

File: java/src/main/java/com/experian/dvs/client/email/validate/VerboseOutput.java
package com.experian.dvs.client.email.validate;

import java.util.HashMap;
import java.util.Map;

public enum VerboseOutput {
    VERIFIED("verified"),
    MAILBOX_DISABLED("mailboxDisabled"),
    MAILBOX_DOES_NOT_EXIST("mailboxDoesNotExist"),
    MAILBOX_FULL("mailboxFull"),
    SYNTAX_FAILURE("syntaxFailure"),
    INTERNATIONAL_CHARACTERS_UNSUPPORTED("internationalCharactersUnsupported"),
    UNREACHABLE("unreachable"),
    ILLEGITIMATE("illegitimate"),
    ROLE_ACCOUNT("roleAccount"),
    TYPO_DOMAIN("typoDomain"),
    LOCAL_PART_SPAM_TRAP("localPartSpamTrap"),
    PROFANITY("Profanity"),
    DISPOSABLE("disposable"),
    UNKNOWN("unknown"),
    TIMEOUT("timeout"),
    ACCEPT_ALL("acceptAll"),
    RELAY_DENIED("relayDenied"),
    BLANK("BLANK");

    private final String name;
    // Map of name to VerboseOutput
    private static final Map<String, VerboseOutput> nameToVerboseOutput = new HashMap<>();
    static {
        for (VerboseOutput output : VerboseOutput.values()) {
            nameToVerboseOutput.put(output.name, output);
        }
    }

    VerboseOutput(final String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public static VerboseOutput fromName(final String name) {
        return nameToVerboseOutput.get(name);
    }
}